IMC_WebDev/MiniCityBlog.md

```diff
@@ -0,0 +1,96 @@
+## Building a Mini City: HTML Portion
+## Purpose
+HTML and CSS are extremely important parts of learning about how websites work. Every website uses HTML in order to display things such as text or images on sites. Being able to grasp a few concepts of  HTML and CSS can help a lot when it comes to learning more intricate subjects in computer science. HTML and CSS are tools used to create and edit websites, and it would be nearly impossible to create a website without using at least some HTML/CSS! For this reason, it is important to get some experience with it!
+
+This tutorial will give you a walkthrough on how certain functions of HTML work and why they are important, all while learning how to build a cool mini-city!
+
+## Setup
+Open any HTML/CSS editor. A good browser-based editor is:
+https://www.w3schools.com/tryit/tryit.asp?filename=tryhtml_default.
+Follow along with this tutorial, and you will be one step closer to building your first mini-city!
+
+## Walkthrough
+
+To build our mini-city, we will need to use tools such as HTML, CSS, and Javascript. This tutorial will go over the HTML portion of the mini-city. You can think of HTML as the foundation of the entire project! If we work to build a strong foundation, our mini-city will look fantastic!
+
+Our first step when using HTML is to establish a “!DOCTYPE html” function. This simply means that all of our code below this will be rendered as an HTML document.
+So, your first line should look like this: <img src=https://lh3.googleusercontent.com/3-aYKYNPyv2Ie-XOajV4_q8EM1PDzeSIxRDKeKtAsSxj_Cy9gSzlhYjDZ_d2UGupdsr5DvKaZy0tfP6NpqwOm8QXHNIIai5XspYBKACo_MODMXnfLni4-VK4KaNcdq2KMLWEGN-A>
+
+
+Got it? Great!
+
+Next, we have to actually create our HTML tag. This just means that we are beginning to write our HTML code, but first we must actually tell our computer that we will be writing in HTML. To do this, simply type “html” on your second line. Your code should look like this so far: <img src=https://lh4.googleusercontent.com/xcemOKcTNUUlo5yAwQvAyAaMhLiwskacjc1LTDy0P0yoaSUpssBCjlURYTfSYsw-yHPBpXGgJCK_Y76lxqLPbjmiEx7WSMxJ5v-OclRHJo1J7YkQNaqI9sR-el_PM8qnaUp0rjvt>
```

Comment:
You don't explain what "tags" are here. Maybe include a brief sentence or two to explain all newly introduced jargon or terminology.

Also, when you say " To do this, simply type “html” on your second line.", I assume you just put html around quotes because using the tag signs made it not show up properly? There's a way to properly show code in Markdown. Check the cheat sheet I linked to above.